src/main/scala/com/wavesplatform/http/BroadcastRoute.scala

```diff
@@ -5,26 +5,39 @@ import com.wavesplatform.network._
 import com.wavesplatform.transaction.{Transaction, ValidationError}
 import com.wavesplatform.utx.UtxPool
 import io.netty.channel.group.ChannelGroup
+import monix.execution.Scheduler
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+private[wavesplatform] object BroadcastRoute {
+  lazy val executionContext = new ExecutionContextExecutor {
```

Comment:
Why isn't the scheduler just instantiated here the way it is done in `ApiMarshallers`?

Reply:
To be compatible with akka-http directive